### ISSUE TITLE
BooleanFilter not work with postgres

### ISSUE THREAD

**Issue by zulus** [sonata-project/SonataDoctrineORMAdminBundle#563]
As in subject. BooleanFilter always produce integer parameter (0 or 1) as the result, even if column is marked as boolean in POJO, so PDO throws:

```
Undefined function: 7 ERROR: operator does not exist: boolean = integer
```
Bundle version: dev-master 3294a3a (as i see in HEAD BooleanFilter didn't change.

**Comment by greg0ire** [comment]
POJO = Plain Old Javascript Object :trollface: ? Can you try a previous version of the bundle and report back? Also, make sure `doctrine/orm` and `doctrine/dbal` are up to date.

**Comment by romainneutron** [comment]
AFAIR The issue is solved in the latest versions of the bundle (^3.0)

**Comment by greg0ire** [comment]
No answer for 10 days. Closing
